src/RangeEnclosures.jl

```diff
@@ -24,7 +24,7 @@ API
 =================#
 include("enclose.jl")
 
-export enclose,
+export enclose, relative_precision,
```

Comment:
I am not sure about exporting. This is rather a utility for comparison, not the typical thing you would want to do.

Reply:
I don't know, it's a pretty nice function, I could imagine someone using `RangeEnclosures` for research would like to use it to compare their algorithms to some reference ones. Well doing `import RangeEnclosures: relative_precision` should not cause carpal tunnel either, so it's not a biggie I guess